.github/workflows/ccpp.yml

```diff
@@ -0,0 +1,41 @@
+name: C/C++ CI
```

Comment:
Should the name be changed to something more specific like ```Darling CI```?

Reply:
changed